src/test/v1.2/support/channels-support-1.2.ts

```diff
@@ -9,28 +9,23 @@ declare let fdc3: DesktopAgent;
 export class ChannelControl1_2 implements ChannelControl<Channel, Context, Listener> {
   private readonly testAppChannelName = "test-channel";
 
-  retrieveAndJoinChannel = async (channelNumber: number): Promise<Channel> => {
-    const channel = await this.getUserChannel(channelNumber);
-    await fdc3.joinChannel(channel.id);
-    return channel;
-  };
-
-  getSystemChannels = async () => {
-    return await fdc3.getSystemChannels();
-  };
+  getNonGlobalUserChannels = async () => {
+    const channels = await fdc3.getSystemChannels();
+    return channels.filter(channel => channel.id.indexOf('global') === -1);
+  }
 
   leaveChannel = async () => {
     return await fdc3.leaveCurrentChannel();
   };
 
-  getUserChannel = async (channel: number): Promise<Channel> => {
+  getNonGlobalUserChannel = async (): Promise<Channel> => {
```

Comment:
can we get this to call getNonGlobalUserChannels() and return the first element of it